libtransmission/resume.cc

```diff
@@ -567,7 +567,12 @@ static auto loadProgress(tr_variant* dict, tr_torrent* tor)
 
         if (std::size(mtimes) != n_files)
         {
-            tr_logAddWarnTor(tor, fmt::format("got {} mtimes; expected {}", std::size(mtimes), n_files));
+            tr_logAddWarnTor(
+                tor,
+                fmt::format(
+                    _("Couldn't load mtime data: expected {expected_size} mtimes; got {actual_size}"),
```

Comment:
I18n-wise:
1. "mtime" will probably lead to questions, especially since it also has to be pluralized somehow
2. this asks for `ngettext()` but "expected N mtimes" and "got N" may need to be pluralized separately in some languages (e.g. Russian); maybe change the sentence structure to avoid that, otherwise will have to split in two phrases which has its own downsides
3. "size" in placeholder names is probably better be "count"

Reply:
Maybe this shouldn't be a warning at all.

The only advantage to translating it is that if there is a bug it's more likely that someone will report it. Other than that, this doesn't inform or help the user -- there's no action they can take and the only harm caused is that the torrent's contents will be lazy-verified during upload when peers request blocks from us.

Demoted to TR_LOG_DEBUG in 582bf02a6ec7cd1b3e59deb909762208af4c0bb1